src/dyn_sensitivity.jl

```diff
@@ -41,12 +41,176 @@ function compute_mefs(P::PowerManagementProblem, net::DynamicPowerNetwork, d, c,
 
     # Construct ∇_x C(x)
     ∇C_dyn = zeros(kkt_dims_dyn(n, m, l, T))
-    idx = 0
-    for _ in 1:T
-        ∇C_dyn[idx+1 : idx+l] .= c
-        idx += static_dim
-    end
-    
+    idx = (t-1)*static_dim
+    ∇C_dyn[idx+1 : idx+l] .= c
```

Comment:
Could this be the bug? I might play with this a bit. Basically, the potential issue here is that the gradient of carbon cost is really dependent on generation at every timestep. If we provide a gradient that only reflects the dependency on a single timestep, we may produce an incorrect gradient with respect to demand.

Reply:
Need to change this.

Reply:
@degleris1 do we agree that `c` is constant over time in our assumptions? And therefore does not need to be specified as an array of size `T`?

Reply:
Agreed! I mean we could model it as varying, but I think it's unnecessary.